### PR TITLE
FBX-487: Bump package version to 5.1.0 (WIP)

### PR DESCRIPTION
**Pull request by windxu88**
## Purpose of this PR:
Bump package version to 5.1.0

**JIRA ticket:**
[FBX-487](https://jira.unity3d.com/browse/FBX-487) 5.1.0 - Open package release PR(s)

**Doc wrap-up** (by @sebastienduverne):
- [x] [FBX-454](https://jira.unity3d.com/browse/FBX-454) - Make FBX Exporter docs main landing page compliant with standards
- [x] User feedback tickets indirectly fixed through the one above: [DOCPA-47](https://jira.unity3d.com/browse/DOCPA-47) + [DOCPA-41](https://jira.unity3d.com/browse/DOCPA-41) + [DOCPA-4](https://jira.unity3d.com/browse/DOCPA-4)
- [x] Rearranged the table of contents for structure consolidation following the recent major edits (previous PRs)
- [x] Fixed navigation links and split/grouped some contents
- [x] Uniformized formatting for notes across the manual
- [x] Uniformized source filenames and removed useless files
- [x] Updated package description in json file for Package Manager display (uniformized with user manual intro)
- [x] Added Changelog entry

### Testing

- [x] Successfully tested the documentation build using Package Doc Tools.

### Note for reviewers

To ease the review process, a built version of the updated doc is [available in Google Drive](https://drive.google.com/drive/folders/1LDepwsWmNClMu72KiKJ68o6MQMtnTAj2?usp=drive_link). To read it, download the `com.unity.formats.fbx@5.1.zip` file, unzip it, and open the `index.html` file in the root folder.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Changes in Fbx Exporter
 
+## [5.1.0] - 2023-09-08
```
Comment (lucillecaillaud) [inline CHANGELOG.md:3]: I guess the entries here could be:
```suggestion
## [5.1.0] - 2023-09-08
### Changed
- Updated the FBX SDK bindings to 5.1.0.
- <Something about the user documentation that Sebastien can help write>
```

**Review by lucillecaillaud — APPROVED** [review]

File: com.unity.formats.fbx/Documentation~/integration.md [hunk 2]
```diff
@@ -4,8 +4,9 @@ Use the Unity Integration tool in **Autodesk® Maya®**, **Autodesk® Maya LT™
...
+| [Set up Unity Integration for your 3D modeling application](integration-setup.md) | Set up Unity Editor integration with Autodesk® Maya®, Autodesk® Maya LT™, or Autodesk® 3ds Max®. |
 | [Use the Unity Integration tool in Maya](integration-maya-roundtrip.md) | Work in Maya to import FBX files from a Unity project and re-export them after editing. |
 | [Manage Export Sets in Maya](integration-maya-manage-export-sets.md) | Create Export Sets to export Maya objects that are not part of an existing Export Set. Edit export options of existing Export Sets. |
 | [Use the Unity Integration tool in 3ds Max](integration-3ds-max-roundtrip.md) | Work in 3ds Max to import FBX files from a Unity project and re-export them after editing. |
 | [Manage objects to export from 3ds Max](integration-3ds-max-manage-objects-to-export.md) | Manage the list of 3ds Max objects to export within Selection Sets. |
+| [Convert Options window](ref-convert-options.md) | Get the description of all settings available in the Convert Options window that displays when you perform a conversion to an FBX Prefab Variant. |
```
Comment (lucillecaillaud) [inline com.unity.formats.fbx/Documentation~/integration.md:12]: This "Convert Options window" reference page is currently NOT a child of this page. Normal or mistake?

**Review by lucillecaillaud — COMMENTED** [review]
Phew, what a wrap-up! Spotted just one odd thing, other than that it looks super good!

File: com.unity.formats.fbx/Documentation~/integration.md (hunk 2 above)
Comment (sebastienduverne) [reply com.unity.formats.fbx/Documentation~/integration.md:12]: Ha! 😅 Clearly a mistake, I don't know what happened, this table line should even be in the Prefab Variants landing page, where it is currently missing. You have better eyes than me – thanks!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
...
```
Comment (sebastienduverne) [inline CHANGELOG.md:3]: Maybe we should update this to a closer date to the actual expected release?

**Review by sebastienduverne — APPROVED** [review]
Approving only to say I'm done with it and get the full green light.
Only a comment about the release date in Changelog (but not critical).

File: CHANGELOG.md (hunk 3 above)
Comment (windxu88) [reply CHANGELOG.md:3]: I can change to today's date which is 2023-09-18 together with another small CI change.

**Pull request merged**